storm-client/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -361,6 +363,7 @@ public void run() {
                             Time.sleep(s);
                     }
                 } catch (Throwable t) {
+                    LOG.info("Async loop Exception Stacktrace is: {} ", getStackTrace(t));
```

Comment:
Why isn't the stack trace available from the log in line 372?

Reply:
The method can return at line 370 without reaching 372. We had instances where users could not understand root cause of the interrupt exception.

Reply:
Thanks. Wouldn't it make sense to add the throwable to the log statement there then, so both L370 and 372 print the exception? (I'm asking why we want a separate log for this stack trace)